addons/host-upgrades/addon.rb

```diff
@@ -1,20 +1,29 @@
 # frozen_string_literal: true
 
 Pharos.addon 'host-upgrades' do
-  version '0.1.0'
+  version '0.0.0'
   license 'Apache License 2.0'
 
   config {
-    attribute :interval, Pharos::Types::String
+    attribute :image_tag, Pharos::Types::String.default("edge")
+    attribute :schedule, Pharos::Types::String
```

Comment:
This absolutely needs to have some regexp for validation, because anyone configuring this is going to be more likely to get the syntax wrong rather than right, and then be left debugging their `CrashLoopBackoff` pods... the feedback cycle for fixing this configuration is too long and painful otherwise.

Reply:
I would prefer to merge this sooner than later.. if regexp takes time then we should open a new issue and mark it to the 1.2.0 milestone.

Reply:
No need, I already have something, it just needs specs to test it.